src/configs/mealType.ts

```diff
@@ -0,0 +1,17 @@
+export type MealType = 'main course' | 'side dish' | 'dessert' | '';
+export enum MealTypeMap {
+  'main course' = 'main course',
```

Comment:
В енамах ключи не должны содержать пробелы, пишем кэмел-кейсом, кавычки убираем
```suggestion
  mainCourse = 'main course',
```

Reply:
done